content/livesync/how-livesync-works.textile

```diff
@@ -0,0 +1,95 @@
+---
+title: How LiveSync works
+meta_description: "LiveSync enables you to synchronize changes in your database to frontend clients at scale."
+product: livesync
+---
+
+LiveSync is a powerful real-time data synchronization feature designed to facilitate seamless collaboration and data consistency across distributed systems. Leveraging Pub/Sub Channels, LiveSync ensures that data updates are propagated instantly and reliably to all connected clients in real-time. By providing bidirectional synchronization capabilities, LiveSync enables developers to effortlessly synchronize data between clients and servers, as well as between different clients, regardless of their location or network conditions.
+
+LiveSync benefits applications requiring real-time collaboration, such as collaborative editing tools, multiplayer games, or financial trading platforms, where maintaining data consistency and low latency are paramount. Developers can build highly responsive and interactive applications that deliver an exceptional real-time user experience.
+
+h2(#channel-based-broadcasting). Channel-based broadcasting
+
+The mechanism for synchronizing data updates across devices using Ably's LiveSync closely mirrors the delivery model of real-time messages to subscribers via Ably's "pub/sub channels":https://ably.com/docs/channels. When a message is published on an Ably channel, it is immediately broadcasted to all subscribers of that channel. Similarly, with LiveSync, data updates added to the outbox table are then published to all subscribed clients in real-time, ensuring seamless data synchronization between the database and all clients.
+
+h2(#frontend). Frontend
+
+For all clients to receive realtime updates of the database table using LiveSync, the frontend data models are used. This uses the Models SDK to carry out the following steps:
```

Comment:
"For all clients to receive realtime updates of the database table using LiveSync, the frontend data models are used." - not sure about this sentence, I would perhaps say e.g. "Clients consume realtime updates to the database table through the Models SDK, which carries out the following steps"